### ISSUE TITLE
作者你好，请问Dice_loss里的tn怎么写呢

### ISSUE THREAD

**Issue by zxd1874** [bubbliiiing/unet-pytorch#6]
tp = torch.sum(temp_target[...,:-1] * temp_inputs, axis=[0,1])
fp = torch.sum(temp_inputs                       , axis=[0,1]) - tp
fn = torch.sum(temp_target[...,:-1]              , axis=[0,1]) - tp

**Comment by bubbliiiing** [comment]
Dice_loss不就是里面这样写吗……

**Comment by zxd1874** [comment]
> Dice_loss不就是里面这样写吗……
对Dice_loss没错，但是我想知道true negative怎么写，麻烦了

**Comment by bubbliiiing** [comment]
torch.sum((1-temp_target[...,:-1]) *(1 - temp_inputs), axis=[0,1])你觉得这样对不对

**Comment by zxd1874** [comment]
作者很抱歉这么晚回复你，我试了一下，这么写应该是对的
